Update download filename to new naming convention

bethcollins92 opened the pull request:
## Done

- Add new fields to modal - Product, Partner, Message and Language
- Include these along with Style and Size to the download filename
- Make small adjustments to alignment of content and text size to account for longer words and titles 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8057
- Fill out all fields in modal to build banners
- Download each size of banner
- Check the filename follows the naming convention mentioned [here](https://github.com/canonical-web-and-design/web-squad/issues/3829#issue-809170576)

## Issue / Card

Fixes [#3829](https://github.com/canonical-web-and-design/web-squad/issues/3829)

webteam-app commented:
Demo starting at https://tools-demos-haus-44.demos.haus

waynecrosby commented:
@bethcollins92 I think most of them are working well, I would just reduce the size of the title font and increase it's line-height a little on the 2 below as currently it's too tight, particularly the Email banner:
<img width="683" alt="Screenshot 2021-02-24 at 14 44 43" src="https://user-images.githubusercontent.com/42238722/109017460-1b0b1e80-76af-11eb-8937-bfbb6d043fb1.png">

<img width="853" alt="Screenshot 2021-02-24 at 14 45 08" src="https://user-images.githubusercontent.com/42238722/109017452-18102e00-76af-11eb-99c6-01ad539a30cf.png">

bethcollins92 commented:
Thanks @waynecrosby, made those changes 👍🏼

pmahnke commented:
@bethcollins92 A few things:

1. On the form, perhaps add a heading above.
![image](https://user-images.githubusercontent.com/441217/109029636-f452e500-76ba-11eb-96b0-d48fb8674d0e.png)

2. Ad banner (160px × 600px) the image isn't being downsized enough?
![image](https://user-images.githubusercontent.com/441217/109029805-1e0c0c00-76bb-11eb-9e23-624fca8902c9.png)

3. Some fonts are a tiny bit fuzzy... perhaps their size needs to be rounded to a closer integer?  I think it is like this on live as well.

waynecrosby commented:
@bethcollins92 Thanks for that! This is the job that keeps on giving! There are a few little niggles, I know we are going to have edge cases and the copy will need to be written to adhere to some sort of guidelines but if you could take a look at the below:

When there is only one logo can we add a variable to move all the copy up so that there is isn't such a big gap between it and the logo? Also, the illustration seems to be expanding out of the bounds of the banner, can you reduce it a little please?
<img width="326" alt="Screenshot 2021-02-24 at 15 21 40" src="https://user-images.githubusercontent.com/42238722/109025580-c1a6ed80-76b6-11eb-9521-2f628234f963.png">

Can you move the illustration down when there are 2 logos, when there is just one it's fine, but it's too close to the MS Azure logo at the moment.
<img width="1177" alt="Screenshot 2021-02-24 at 15 21 03" src="https://user-images.githubusercontent.com/42238722/109025591-c4a1de00-76b6-11eb-983f-83dc73888911.png">

Also, when there are 2 lines in the headline, can we add a variable to move all the copy up a bit? I think Peter built in some of these in the previous iteration, but if it's going to be too onerous don't worry.
<img width="832" alt="Screenshot 2021-02-24 at 15 20 40" src="https://user-images.githubusercontent.com/42238722/109025600-c5d30b00-76b6-11eb-981f-f69a4233f6ba.png">

bethcollins92 commented:
@pmahnke I've added the suggested text, and fixed the size of the image on the 160X60. I've spoken to @steverydz, not sure about the blurry text as they are whole integers, think it might be a canvas thing. 

@waynecrosby I've moved the image down slightly when there is both logos on the 1200x628, and moved the text up on the 160X60. Amber needs to use these asap so I'm going to make another issue for the text wrapping on the email banner.

pmahnke reviewed (approved):
+1

waynecrosby commented:
@bethcollins92 Thanks for updating!
